pkg/api/config/v1alpha1/config_types.go

```diff
@@ -78,6 +78,10 @@ type IngressConfig struct {
 	// PathType defines how ingress paths should be interpreted.
 	// Allowed values: Exact, Prefix, ImplementationSpecific
 	PathType networkingv1.PathType `json:"pathType"`
+
+	// SkipTLS disables TLS configuration for ingress resources. This is useful
+	// if a 3rd-party component is handling the HTTPS TLS termination and certificates.
+	SkipTLS bool `json:"skipTls"`
```

Comment:
What is the convention with JSON names for capitalised abbreviations? Should it be `skipTls` or `skipTLS`?